internal/rest/mgmt/helpers.go

```diff
@@ -77,7 +87,157 @@ func ServicePolicyFromFilter(client *rest_management_api_client.ZitiEdgeManageme
 	params.SetTimeout(5 * time.Second)
 	resp, err := client.ServicePolicy.ListServicePolicies(params, nil)
 	if err != nil {
-		log.Errorf("Could not obtain an ID for the service with filter %s: %v", filter, err)
+		log.Errorf("Could not obtain an ID for the service policy with filter %s: %v", filter, err)
+		return nil
+	}
+	if resp == nil || resp.Payload == nil || resp.Payload.Data == nil || len(resp.Payload.Data) == 0 {
+		return nil
+	}
+	return resp.Payload.Data[0]
+}
+
+func AuthPolicyFromFilter(client *rest_management_api_client.ZitiEdgeManagement, filter string) *rest_model.AuthPolicyDetail {
+	params := &auth_policy.ListAuthPoliciesParams{
+		Filter:  &filter,
+		Context: context.Background(),
+	}
+	params.SetTimeout(5 * time.Second)
```

Comment:
these `5 * time.Second` should at very least references a single constant. might be worthwhile making it configurable some day (not now)